File: beecrowd/1/3047/3047.py
def max(A, B, C):
    max = A
    if B > max:
        max = B
    if C > max:
        max = C

    return max


M = int(input())
A = int(input())
B = int(input())
C = M - (A + B)
n = max(A, B, C)
print(n)

File: beecrowd/1/3047/3047.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

function max(A, B, C) {
    let m = A;
    if (B > m) {
        m = B;
    }
    if (C > m) {
        m = C;
    }
    return m;
}

let M, A, B;
M = parseInt(lines.shift());
A = parseInt(lines.shift());
B = parseInt(lines.shift());
let C = M - (A + B);
let n = max(A, B, C);
console.log(n);
